### PR TITLE
Update onboarding.md

### PR DESCRIPTION
**Pull request by its-a-lisa-at-work**
closes https://github.com/18F/tts-tech-portfolio/issues/491 and supports https://github.com/18F/tts-tech-portfolio/pull/810 which is now out of date & out of sync with this; but will make updates there once this gets approved

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/onboarding.md [hunk 1]
```diff
@@ -42,17 +34,43 @@ Learn our policies and procedures:
...
 
 - [ ] [TTS Software](https://groups.google.com/a/gsa.gov/forum/#!managemembers/tts-software/add)
 - [ ] [TTS Technology Portfolio](https://groups.google.com/a/gsa.gov/forum/#!managemembers/devops/add)
 - [ ] [TTS Vulnerability Reports](https://groups.google.com/a/gsa.gov/forum/#!managemembers/tts-vulnerability-reports/add)
+
+### Account Access- User
+- [ ] Issue AWS sandbox accounts
+
+### Account Access - Admin
+- [ ] Provide Root Credentials
```
Comment (afeld) [inline .github/ISSUE_TEMPLATE/onboarding.md:59]:
```suggestion
- [ ] Add as an admin
```

File: .github/ISSUE_TEMPLATE/onboarding.md [hunk 2]
```diff
@@ -42,17 +34,43 @@ Learn our policies and procedures:
...
+- [ ] Federalist account
 
 ### Google Groups
 
 - [ ] [TTS Software](https://groups.google.com/a/gsa.gov/forum/#!managemembers/tts-software/add)
 - [ ] [TTS Technology Portfolio](https://groups.google.com/a/gsa.gov/forum/#!managemembers/devops/add)
 - [ ] [TTS Vulnerability Reports](https://groups.google.com/a/gsa.gov/forum/#!managemembers/tts-vulnerability-reports/add)
+
+### Account Access- User
+- [ ] Issue AWS sandbox accounts
```
Comment (afeld) [inline .github/ISSUE_TEMPLATE/onboarding.md:56]:
```suggestion
- [ ] Create AWS IAM user for [cross-account access](https://github.com/18F/aws-admin#cross-account-access)
```

File: .github/ISSUE_TEMPLATE/onboarding.md [hunk 3]
```diff
@@ -42,17 +34,43 @@ Learn our policies and procedures:
...
+  - [ ] DockerHub
+  - [ ] Microsoft Office
+  - [ ] HackerOne
+
+### Give Access to Google Forms
+- [ ] Anonymous Feedback
+- [ ] Vulnerability Response
+
+### Learn about what we do
+- [ ] [DNS](https://github.com/18F/dns
```
Comment (afeld) [inline .github/ISSUE_TEMPLATE/onboarding.md:75]:
```suggestion
- [ ] [DNS](https://github.com/18F/dns)
```

File: .github/ISSUE_TEMPLATE/onboarding.md [hunk 4]
```diff
@@ -42,17 +34,43 @@ Learn our policies and procedures:
...
+  - [ ] Trello
+  - [ ] GitHub
+  - [ ] NewRelic
+  - [ ] InVision
+  - [ ] Mural
+  - [ ] DockerHub
+  - [ ] Microsoft Office
+  - [ ] HackerOne
+
+### Give Access to Google Forms
```
Comment (afeld) [inline .github/ISSUE_TEMPLATE/onboarding.md:70]: If they are shared with devops@gsa.gov, granting access shouldn't be necessary.

```suggestion
### Subscribe to Google Forms
```

File: .github/ISSUE_TEMPLATE/onboarding.md [hunk 5]
```diff
@@ -1,11 +1,3 @@
----
-name: New team member
-about: Checklist for onboarding someone to the Tech Portfolio team
-title: ""
-labels: "g: accepted, i: misc"
-assignees: ""
----
```
[the remark this answers is not in the document]
Comment (its-a-lisa-at-work) [reply .github/ISSUE_TEMPLATE/onboarding.md:7]: nope! good catch

File: .github/ISSUE_TEMPLATE/onboarding.md (hunk 4 above)
Comment (its-a-lisa-at-work) [reply .github/ISSUE_TEMPLATE/onboarding.md:70]: Hmm, I was thinking that someone who had access for Google Forms that we've created would have to share permissions so the folks are able to see past responses and edit the form if ever needed..

**Review by afeld — APPROVED** [review]

**Pull request merged**